### PR TITLE
Switch Rules Admin tool text editor to load parsed text from the backend

### LINKED ISSUE #432: Gtk Warning about colors on policy analysis view
Open an event log in policy analysis view and navigate to the object view.  If there are denies present you get a warning on the console

```
(__main__.py:13341): Gtk-WARNING **: 00:52:01.505: Failed to set text from markup due to error parsing markup: Value of 'foreground' attribute on <span> tag on line 1 could not be parsed; should be a color specification, not 'FF0000'
```

### PR DESCRIPTION
**Pull request by dorschs57**
Switched the Rules Administer tool's text editor back to loading the text from the backend. As of #439 the text is now available there as a single string of text that is parsed and has file markers added. As part of this change the text editor now loads the GTK SourceView language file for fapolicyd rules. This gives the text editor code highlighting and other enhancements.

Also fixed the warnings that were coming from the red text config setting.

closed #432

### CODE REVIEW

File: fapolicy_analyzer/ui/rules/rules_text_view.py [hunk 1]
```diff
@@ -13,27 +13,34 @@
...
-                text = f.read()
-        except (IOError, FileNotFoundError) as e:
-            logging.error("%s: %s", RULES_FILE_READ_ERROR, e)
-            dispatch(add_notification(RULES_FILE_READ_ERROR, NotificationType.ERROR))
+        # add fapolicyd rules lang file
+        self.manager = GtkSource.LanguageManager.get_default()
+        self.manager.set_search_path(
+            [
+                *self.manager.get_search_path(),
+                "/home/addorschs/fapolicy-analyzer/share/sourceview/language-specs/",
```
Comment (jw3) [inline fapolicy_analyzer/ui/rules/rules_text_view.py:35]: I'm guessing this is still a WIP, but just in case Ill note this line...

Comment (dorschs57) [reply fapolicy_analyzer/ui/rules/rules_text_view.py:35]: lol, yes that was a late Friday push to get done. I'll clean that up today.

**Comment by jw3** [comment]
The best solution to fix the RPM build is to remove the spec entries for the lang and style.

I was planning on having those be part of our RPM install, but like the resource approach in this PR better for now.

It might make sense to move those files under `fapolicyd_analyzer/resources` as well.

File: fapolicy_analyzer/ui/rules/rules_text_view.py (hunk 1 above)
Comment (dorschs57) [reply fapolicy_analyzer/ui/rules/rules_text_view.py]: Fixed up. It now loads the lang and style files from resources.

**Comment by jw3** [comment]
Git should have preserved the history of the lang and style when you moved it, doesnt look like it did.

File: scripts/rpm/fapolicy-analyzer.spec.j2 [hunk 2]
```diff
@@ -51,8 +51,6 @@ VERSION=%{app_version} %{python3} setup.py bdist_wheel
 %{python3} -m pip install --no-deps --no-input -v --target %{buildroot}%{python3_sitelib} dist/*.whl
 %{python3} -m pip install --no-deps --no-input -v --target %{buildroot}%{python3_sitelib} vendor/ReduxPY
 install bin/fapolicy-analyzer %{buildroot}%{_sbindir}/fapolicy-analyzer -D
-install share/sourceview/language-specs/fapolicyd-rules.lang %{buildroot}%{_datadir}/gtksourceview-3.0/language-specs/fapolicyd-rules.lang -D
-install share/sourceview/styles/fapolicyd.xml %{buildroot}%{_datadir}/gtksourceview-3.0/styles/fapolicyd.xml -D
```
Comment (jw3) [inline scripts/rpm/fapolicy-analyzer.spec.j2:55]: You'll need to remove the sourceview entry under `%files` below.

Comment (dorschs57) [reply scripts/rpm/fapolicy-analyzer.spec.j2:55]: Just found that. Thx.

**Review by jw3 — APPROVED** [review]
Looks good

**Pull request merged**